docs/source/development/contributing.rst

```diff
@@ -0,0 +1,223 @@
+=====================
+Contributing to Eland
+=====================
+
+Eland is an open source project and we love to receive contributions
+from our community — you! There are many ways to contribute, from
+writing tutorials or blog posts, improving the documentation, submitting
+bug reports and feature requests or writing code which can be
+incorporated into eland itself.
+
+Bug reports
+-----------
+
+If you think you have found a bug in eland, first make sure that you are
+testing against the `latest version of
+eland <https://github.com/elastic/eland>`__ - your issue may already
+have been fixed. If not, search our `issues
+list <https://github.com/elastic/eland/issues>`__ on GitHub in case a
+similar issue has already been opened.
+
+It is very helpful if you can prepare a reproduction of the bug. In
+other words, provide a small test case which we can run to confirm your
+bug. It makes it easier to find the problem and to fix it. Test cases
+should be provided as python scripts, ideally with some details of your
+Elasticsearch environment and index mappings, and (where appropriate) a
```

Comment:
change to opensearch?

Reply:
I'll delete this file in the next PR (mistakenly uploaded). I'm referring to our [Contributing.md](https://github.com/opensearch-project/opensearch-py-ml/blob/main/CONTRIBUTING.md)